config/samples_2018.yaml

```diff
@@ -15,6 +15,7 @@ GLOBAL:
     - Flag_ecalBadCalibFilter
   triggerFile: config/triggers_2018.yaml
   crossSectionsFile : config/crossSections13TeV.yaml
+  lumiFile: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions18/13TeV/Legacy_2018/Cert_314472-325175_13TeV_Legacy2018_Collisions18_JSON.txt
```

Comment:
Just for my knowledge, is there a UltraLegacy directory?
I see different types of certification files and log files. Why did you choose this?

Reply:
this is UL json. The recommendations from the LUMI POG can be found here: https://twiki.cern.ch/twiki/bin/view/CMS/LumiRecommendationsRun2

Reply:
Thank you!